Main section

VitaliiKarpovets opened the pull request.

VitaliiKarpovets committed:
first step

Vladosik12333 commented:
На Stund Up.

Vladosik12333 reviewed (dismissed):
На Stund Up.

VolodymyrTsyhylyk committed:
html+css шаблона 1 картки

VitaliiKarpovets committed:
main html+css gallery section + spinner html+scc

VitaliiKarpovets committed:
Merge branch 'feature/gallery-html' of https://github.com/Vladosik12333/project-4-filmoteka into feature/gallery-html

Yurii97 commented on src/templates/card-gallery.hbs:
```diff
@@ -1 +1,17 @@
-<div></div>
\ No newline at end of file
+{{#each this}}
+  <li class='card'>
```
добавити дата атрибут! Обговорювали після зборів

Yurii97 commented on src/sass/components/_gallery.scss:
```diff
@@ -0,0 +1,89 @@
+.section {
+  text-align: center;
+  background-color: #ffffff;
```
вставляємо колір зі змінних :  

// акцентний помаранчевий:
$accentColort: #ff6b08;

// текст
$textLightColor: #ffffff;
$textDarkColor: #000000;

// підкладка під фон:
$bgOverlayColor: rgba(0, 0, 0, 0.56);

// фон альтернативний (футер і кнопки світчера)
$bgSecondaryColor: #f7f7f7;

Yurii97 commented on src/sass/components/_gallery.scss:
```diff
@@ -0,0 +1,89 @@
+  width: 2em;
+  height: 2em;
+  vertical-align: text-bottom;
+  background-color: currentColor;
```
???
// акцентний помаранчевий:
$accentColort: #ff6b08;

// текст
$textLightColor: #ffffff;
$textDarkColor: #000000;

// підкладка під фон:
$bgOverlayColor: rgba(0, 0, 0, 0.56);

// фон альтернативний (футер і кнопки світчера)
$bgSecondaryColor: #f7f7f7;

Yurii97 reviewed (commented):
Внести правки

VitaliiKarpovets committed:
fixed template, and styles for gallery card-gallery

в gallery.html поставив спінер після списка фільмів
переробив шаблон
переписав стилі для шаблону

rostunho reviewed (approved):
Беремо для тестування живої сторінки. Рефакторимо баг-фіксами.
